src/accordion/accordion.ts

```diff
@@ -216,7 +239,18 @@ export class NgbAccordion implements AfterContentChecked {
    */
   @Output() panelChange = new EventEmitter<NgbPanelChangeEvent>();
 
-  constructor(config: NgbAccordionConfig) {
+  /**
+   * An event emitted when a panel is shown, after the transition. The payload is the panel id.
```

Comment:
Could we be more specific, please ? meaning that "when the panel is shown" is not very clear...

Maybe `An event emitted when the expanding animation is finished on the panel` ?